doc/guide/install.rst

```diff
@@ -1,9 +1,9 @@
 ********************
-sasmodels Setup
+Sasmodels setup
```

Comment:
To follow the convetion. Shouldn't "Setup" also start with capital letter?

Reply:
Done